### DISCUSSION TITLE
How to enable CPU overclocking?

### DISCUSSION THREAD

**Discussion by DimaSergeew** [BartoszCichecki/LenovoLegionToolkit#922] [Q&A] [answered]
How to enable CPU overclocking in Lenovo Toolkit? It's available for me in Lenovo Vantage, but it's not in Lenovo Toolkit.

**Comment by BartoszCichecki** [comment, marked as answer]
There is no CPU OC in LLT. Vantage uses Intel XTU for OC, so you can install it separately or (better) use ThrottleStop for OC/UV.
